pyproject.toml

```diff
@@ -23,6 +23,7 @@ isort = "^5.12.0"
 pyupgrade = "^3.3.1"
 codespell = "^2.2.2"
 pytest-cov = "^4.0.0"
+nuitka = "^1.5.4"
```

Comment:
Po co to jest???

Reply:
Do budowania .exe by gosc nie musial niczego instalowac

Reply:
Jak wejdziesz w https://github.com/wvffle/ATS-project-simple/actions/runs/4600151430 to zobaczysz ze na dole mozesz pobrac plik .exe